lib/cli/src/generators/HTML/index.js

```diff
@@ -9,7 +9,13 @@ import {
 import { STORY_FORMAT } from '../../project_types';
 
 export default async (npmOptions, { storyFormat }) => {
-  const packages = ['@storybook/html'];
+  const packages = [
+    '@storybook/html',
+    '@storybook/addon-actions',
+    '@storybook/addon-links',
+    '@storybook/addons',
```

Comment:
can we check if `@storybook/addons` is necessary?

Reply:
Not needed, see https://github.com/storybookjs/storybook/pull/10703#issuecomment-628227892